content/account/organizations.textile

```diff
@@ -0,0 +1,178 @@
+meta_keywords: "Ably organizations, user provisioning, SSO, SCIM, access management, Ably dashboard, account roles, permissions"
+---
+
+Use organizations to manage multiple Ably accounts by centralizing user access and permissions under a single organizational structure. Organizations streamline user provisioning through a Single Sign-On (SSO) configuration and utilize System for Cross-domain Identity Management (SCIM) for group-based access control.
+
+Organizations enable you to assign roles and permissions across different accounts, such as production, staging, and development environments.
+
+<aside data-type='note'>
+<p>An "Enterprise":/pricing/enterprise account is required to use organizations. "Contact us":https://ably.com/contact to enable organizations for your account.</p>
+</aside>
+
+h2(#primary). Primary account
+
+The primary account is an organization's main account and holds the following privileges:
+
+* The highest level of access to the organization.
+* Ownership of all all accounts within the organization.
+* The default account used for provisioning.
+
+h2(#accounts). Create accounts in an organization
+
+Create "accounts":https://ably.com/accounts/any in an organization:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Accounts*.
+** Click *New account*.
+** Add an account name and *Create account*.
+
+h3(#isolate). Isolate environments
+
+Use separate accounts within an organization to create isolated environments, such as production, staging, and development. Assign each environment a "package":/pricing#packages that meets its specific needs. For example, production may need high capacity with an Enterprise package, staging might use a Standard package, and development a Free package.
+
+h2(#manage). Manage user account access
+
+"Managing user":/account/users account access allows control of both organizational management permissions and access to specific accounts within your organization.
+
+Manage user account access:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Users* to view the list of users.
+** Select the *user* whose access you want to manage.
+** Specify the user's required *Permissions* for the account.
+** Assign the necessary *Roles* to grant the user access to specific *Accounts*.
+** Click *Back to Organization users* to save your changes.
+* Add the user to a new account.
+** Enter the account name.
+** Click *Create Account*.
+
+<aside data-type='note'>
+<p>Users provisioned through SCIM cannot modify their name or email address within Ably. All personal information updates must be made through your identity provider, and changes will sync to Ably upon the next SCIM update cycle.</p>
+</aside>
+
+h3(#permissions). User permissions
+
+A user's total permissions consist of:
+
+* Roles and permissions assigned directly to the user.
+* Roles and permissions granted to all groups the user is part of.
```

Comment:
I'm not sure this needs to be a sub-heading as it contradicts with the 

Also, where are groups assigned permissions, as this is the first place they are mentioned.